performance-test/tasks/interest_tasks.py

```
@@ -0,0 +1,42 @@
+    self.client.get(
+        "/api/interests",
+        headers=self.user.get_headers(),
+        params={
+          "orderBy": random.choice(["name", "subscriberCount"]),
+          "direction": random.choice(["ASC", "DESC"]),
+          "limit": 20
+        }
+    )
+
+    @task(3)
+    def get_sorted(self):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `get_sorted`

Add return type annotation: `None`

(ANN202)